test/spec/provider/bpmn/MessageProps.spec.js

```diff
@@ -259,38 +259,6 @@ describe('provider/bpmn - MessageProps', function() {
 
     });
 
-
-    describe('show error', function() {
```

Comment:
Wouldn't it still make sense to have such an integration test? Maybe not for all specific props, but a general one (e.g. in the BpmnPropertiesPanelRenderer tests)?

Reply:
Alright, I'll try and add one.

Reply:
Thanks!